app/models/user.rb

```diff
@@ -1,7 +1,7 @@
 class User < ApplicationRecord
-  has_many :posts, foreign_key: 'author_id'
-  has_many :comments, foreign_key: 'author_id'
-  has_many :likes, foreign_key: 'author_id'
+  has_many :posts, class_name: 'Post', foreign_key: 'author_id'
+  has_many :comments, class_name: 'Comment', foreign_key: 'post_id'
+  has_many :likes, class_name: 'Like', foreign_key: 'post_id'
 
```

Comment:
- [ ] Kindly change the `foreign_key:` `author_id`. Remember that the `User` is connected to the tables through `author_id`. When I try to create `comments` and `likes,` I get that error. Kindly fix it. 👍

Reply:
Hello @Gambit142 so do you mean I should change the foreign_key `post_id` to `author_id`?

Reply:
Hello @oluyaratosin123, yes, kindly change it from `post_id` to `author_id`. 👍